### ISSUE TITLE
Remove RandomSplitter from notebooks

### ISSUE THREAD

**Issue by ai-fast-track** [airctic/icevision#432]
## 📓 Documentation Update
RandomSplitter is now passed by default. Therefore, we need to update the notebooks, in the [notebooks folder](https://github.com/airctic/icevision/tree/master/notebooks).

1- The following line should be removed:
```
# Randomly split our data into train/valid
data_splitter = RandomSplitter([0.8, 0.2])
```

2- The following line should be modified: remove `data_splitter` from the `parser.parse()` call
**Before:**
```
train_records, valid_records = parser.parse(data_splitter)
```

**After:**
```
train_records, valid_records = parser.parse()
```

**Comment by singhalpranav22** [comment]
Can i do this work ?

**Comment by ai-fast-track** [comment]
Yes of course you can work on it. We also have a similar issue here #431 

Please, let us know if you need some help.

**Comment by ai-fast-track** [comment]
Closed in PR #446
